src/vs/workbench/contrib/inlineChat/browser/inlineChatDecorations.ts

```diff
@@ -104,6 +93,10 @@ export class InlineChatDecorationsContribution extends Disposable implements IEd
 		this._gutterDecorationOpaque.glyphMarginHoverMessage = hoverMessage;
 	}
 
+	private _updateCurrentBreakpoints(uri: URI) {
+		this._currentBreakpoints = [...this._debugService.getModel().getBreakpoints({ uri })];
```

Comment:
Is it necessary to make a copy?

Reply:
The breakpoints array returned by getBreakpoints is readonly so it can not be assigned to this._currentBreakpoints which is not readonly

Reply:
The array needs to be read only, but the property an still be modified:
`private _currentBreakpoints: readonly IBreakpoint[] = [];`

Reply:
Ah right I misunderstood then, thanks for pointing it out